sympy/physics/mechanics/tests/test_linearize.py

```diff
@@ -81,7 +81,7 @@ def test_linearize_rolling_disc_kane():
     linearizer = KM.to_linearizer()
     assert linearizer.f_c == f_c
     assert linearizer.f_v == f_v
-    assert linearizer.f_a == f_v.diff(t)
+    assert linearizer.f_a == f_v.diff(t).subs(KM.kindiffdict())
```

Comment:
This test was necessary to change, as the prior version was outputting an incorrect result.